DreamJournal/DreamJournal/NewEntryView.swift

```diff
@@ -0,0 +1,206 @@
+//
+//  ContentView.swift
+//  DreamJournal
+//
+//  Created by CUBS Customer on 10/7/22.
+//
+
+import SwiftUI
+import CoreData
+
+struct CustomColor {
+    static let Navy = Color("Navy")
+    static let SkyPurple = Color("SkyPurple")
+    static let LavenderBox = Color("LavenderBox")
+}
+
+enum Category: String, Identifiable, CaseIterable {
+    var id: UUID {
+        return UUID()
+    }
+    
+    case nightmare = "Nightmare"
+    case neutral = "Neutral Dream"
+    case good = "Good Dream"
+}
+
+extension Category {
+    var title: String {
+        switch self {
+            case .nightmare:
+                return rawValue
+            case .neutral:
+                return rawValue
+            case .good:
+                return rawValue
+        }
```

Comment:
I'm guessing this maybe stuck around after a refactor or something, but seems like you should just return `rawValue` instead of doing this whole switch!

Reply:
That was definitely my bad-- I understand the raw values better now so I know to get rid of the switch case! I'll fix it ASAP!